src/services/routes/Local.js

```diff
@@ -8,7 +8,7 @@ import RNFS from 'react-native-fs';
 export async function readDirectory(dirpath) {
   return RNFS.readdir(dirpath)
```

Comment:
Hi Helena, I think we'll need to rehaul the RNFS call here, sorry it will likely change the rest of your code but should be an easy fix. Essentially, we need to grab the last modified date of the file in order to sort the lesson plans in the Library by last edited (also display it in the [options menu)](https://www.figma.com/file/kSZduD0A5hk8sHmvqiC7DE/Sistema-Hi-Fidelity-Designs-%2B-Design-System?node-id=639%3A18072&t=x27PBltneYiqUkgE-0). This means, we need to change this RNFS call to [RNFS.readDir](https://github.com/itinance/react-native-fs#readdirdirpath-string-promisereaddiritem), which will return `Promise<ReadDirItem[]>`. 

Here's what you can follow along:
1. In `Local.js`, return `ReadDirItem[]` without any modifications.
2. Inside `LessonPlanService.js`, map the result and extract the info. For instance, 
```javascript
// where result is ReadDirItem[]
const lpInfo = result.map((dirItem: ReadDirItem) => {
    // do whatever manipulation, maybe like turn the date into DateTime object
    return { mtime: date,     // The last modified date of the file
                 name: string }     // The name of the item
})
```
You might even want to return the array type of `lpInfo` into a model in`model.js`, so that it's clear on the frontend what parameters are in the object you're getting.

Reply:
I fixed this, instead of changing readDirectory (since i use it in a different funciton), I added a new method in local for readDir called readDDirectory.